pyproject.toml

```diff
@@ -17,6 +17,7 @@ license = {file = "LICENSE"}
 requires-python = ">=3.10"
 dependencies = [
     "fastapi>=0.68.0",
+    "starlette",
```

Comment:
`starlette` is a hard dependency of `fastapi`. Meaning, this just makes it explicit that we are also using `starlette` code, but the total number of dependencies stays the same.

Reply:
Sounds good!